.eslintrc.js

```diff
@@ -15,6 +15,10 @@ module.exports = {
         '@jitsi/eslint-config',
         '@jitsi/eslint-config/typescript'
     ],
+    "rules": {
```

Comment:
Tiny comment: if this rule makes sense as a general rule, let's have it in the eslint repo?